pkgs/applications/audio/reaper/default.nix

```diff
@@ -19,12 +19,12 @@
 
 stdenv.mkDerivation rec {
   pname = "reaper";
-  version = "6.61";
+  version = "6.64";
 
   src = fetchurl {
     url = "https://www.reaper.fm/files/${lib.versions.major version}.x/reaper${builtins.replaceStrings ["."] [""] version}_linux_${stdenv.hostPlatform.qemuArch}.tar.xz";
     hash = {
-      x86_64-linux = "sha256-Lp2EVky1+ruc86LdMmvhZIisoYl0OxdkVnN3h/u09IQ=";
+      x86_64-linux = "sha256-pg4cqhfIhtHKW4x/p/QTeZBqVIZW6DbsHFRvqt8fhNQ=";
       aarch64-linux = "sha256-sPLCMA//xAdWXjY7++R6eLWS56Zi0u+9ju7JlICGvVc=";
```

Comment:
This is not updated.

Reply:
Well spotted,  thanks.  I'll open a new PR with full update. May need to fix the updater too  🤔

Reply:
Fixed the updater in https://github.com/NixOS/nixpkgs/pull/180080